lib/tests/streamlit/delta_generator_test.py

```diff
@@ -325,6 +325,11 @@ def test_bad_columns(self):
         with self.assertRaises(StreamlitAPIException):
             st.beta_columns([1, 0, -1])
 
+    def test_nested_columns(self):
+        level1, _ = st.beta_columns(2)
+        with self.assertRaises(StreamlitAPIException):
+            level2, _ = level1.beta_columns(2)
+
```

Comment:
Should probably add a test for nested expanders, right?

Reply:
Done!

(My philosophy on tests tends to lean "less is more", with an emphasis on catching breakages, so which is why I didn't bother at first; but these are short, and I'm happy to defer to reviewer here.)